geonode/layers/models.py

```diff
@@ -164,6 +164,12 @@ def is_vector(self):
     def is_raster(self):
         return self.subtype == "raster"
 
+    @property
+    def supports_time(self):
+        if self.is_vector():
```

Comment:
@Gpetrak I would also check if suitable attributes ("xsd:dateTime", "xsd:date") are available for this dataset. See also the other comment on this, about listing the time attributes with a method on the Dataset model

Reply:
@giohappy done